src/open_inwoner/templates/pages/profile/notifications.html

```diff
@@ -9,6 +9,23 @@ <h1 class="h1" id="title">
     {% trans "Kies voor welk onderwerp je meldingen wilt ontvangen" %}
 </p>
 
-{% form form_object=form method="POST" id="change-notifications" submit_text=_("Opslaan") secondary_href='profile:detail' secondary_text=_('Terug') secondary_icon='arrow_backward' secondary_icon_position="before" %}
+<form method="POST" id="change-notifications" action="{% url 'profile:notifications' %}" class="form" novalidate>
+    {% csrf_token %}
+
+    {% for field in form.fields %}
+        {% autorender_field form field %}
+    {% endfor %}
+
+    <div class="form__control">
+        {# pseudo checkbox for notifications that cannot be disabled #}
+        <div class="checkbox">
+            <span role="checkbox" aria-disabled="true" aria-checked="true" class="checkbox__input checkbox__pseudo checkbox__input--disabled" aria-labelledby="id_cases_notifications_action_required" tabindex="0"></span>
+            <span class="checkbox__label checkbox__pseudo-label" id="id_cases_notifications_action_required">{% trans "Zaaknotificaties - action required" %}</span>
```

Comment:
`{% trans "Zaaknotificaties - action required" %}` is a mix of Dutch and English (should have been cleaned up as part of my PR).

Reply:
Guess I should make it all in Dutch (including the paragraph)? 
(We have new translations for this one now I can use)

Reply:
Yes, client-facing text should be Dutch, preferably. It will be translated anyways, but this will make it easier and more consistent.